### ISSUE TITLE
Eclipse Plugin just maps Java articles, however, it is possible to scan a PHP Application

### ISSUE THREAD

**Issue by michaelhidalgo** [TeamMentor/TeamMentor_Eclipse_Plugin#31]
As displayed in the image below, when escaning a PHP application, the HP Fortify Plugin shows up Java articles. It turns out that the programming language by default is Java.

We need to see if is it possible to retrieve what programming Language is being use in Eclipse and that way be able to show content based on that language. It means that if I'm scanning a Java application or a PHP application, the articles displayed are the ones belonging to the technology. 

![screen shot 2014-12-23 at 1 30 51 pm](https://cloud.githubusercontent.com/assets/1732035/5542121/fd95dbb4-8aa7-11e4-93bd-e5653a02c23d.png)

**Comment by DinisCruz** [comment]
Yeah that info is available on the object we get from Fortify (in there there are tons of values, some map to language/query-ids)

**Comment by michaelhidalgo** [comment]
Cool, I will put my head around to see how to discover if the language being used is PHP.
